pages/index.tsx

```diff
@@ -60,14 +62,14 @@ export default function Donate({
     async function loadProjects() {
       const currencyCode = getStoredCurrency();
```

Comment:
I assume this line (and the import of getStoredCurrency ) is not necessary anymore, is it?

Reply:
But before you change this, please consider my comment https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/524#discussion_r534021348 below.

Reply:
Yes removed it. and rewrote with context in `planet-webapp/pages/[p].tsx`